Triggers.mediawiki

```diff
@@ -0,0 +1,78 @@
+<pre>
+  NEP: <to be assigned>
+  Title: Triggers for NeoContract
+  Author: Erik Zhang <erik@neo.org>
+  Type: Standard
+  Status: Draft
+  Created: 2017-10-16
+</pre>
+
+==Abstract==
+
+Trigger is a mechanism for triggering execution for smart contracts. This NEP defines 4 kinds of triggers, they are <code>Verification</code>, <code>VerificationR</code>, <code>Application</code> and <code>ApplicationR</code>.
```

Comment:
Since writing "two kinds of triggers " in Section "Rationale", you might wanna change "4 kinds of triggers" to "four different kinds of triggers"